client/src/constants/apiConstants.js

```diff
@@ -16,6 +16,6 @@ export {
   API_SOURCE_SAT_VERSION,
   API_SOURCE_SSL_CERT,
   API_SOURCE_SSL_DISABLE,
-  API_SOURCE_SSL_PROT,
+  API_SOURCE_SSL_PORT,
```

Comment:
yeahhhh, I should have named this one better, it was actually "prot" not "port"... it's an enum based on the API swagger doc...  `SSLv2, SSLv23, SSLv3, TLSv1, TLSv1_1, TLSv1_2`... maybe flip it to `PROTOCOL`

Reply:
ah....